src/sources/host_metrics.rs

```diff
@@ -751,42 +747,57 @@ pub fn init_roots() {
 }
 
 impl FilterList {
-    fn is_empty(&self) -> bool {
-        self.includes.is_none() && self.excludes.is_none()
-    }
-
-    fn contains_str(&self, value: &str) -> bool {
-        (match &self.includes {
+    fn contains_str(&self, value: Option<&str>) -> bool {
+        (match (&self.includes, value) {
```

Comment:
Hmm, it's unfortunate this has to change to support testing, as it is always called with `Some(…)`.

Reply:
Yeah, I thought that too though the parity is nice.